design/FY2019/AirSource_EIR_WaterHeatPump.md

```diff
@@ -0,0 +1,163 @@
+# Adding Air Source Capability to the EIR Water-to-Water Heat Pump
+
+**Matt Mitchell, Edwin Lee - NREL**
+
+- Original NFP: August 30, 2019
+- Revision Date: N/A
+
+## Justification for New Feature
+
+The EIR-formulated water-to-water heat pump model was added with EnergyPlus 9.1.0. This model allows users to evaluate energy calculations on water source plants where a heat pump provides heating and cooling. The EIR formulation is desired by users as the formulation is accepted as an accurate representation of heat pump operation under common conditions.
+
+The water-to-water model is a welcome addition to the EnergyPlus codebase, however an even further desired feature is to collect multiple water-based components and provide them with heating and cooling from an air-source heat pump. As such, this task will modify the current EIR-formulated heat pump and enable the physics of an air-source heat pump.
+
+## Email and Conference Call Conclusions
+
+N/A
+
+## Overview
+
+Since there is an already-existing EIR-WWHP model, this work should integrated with it rather than creating a new object. The current object is:
+
+- HeatPump:WaterToWater:EIR:(Cooling/Heating)
+
+This model is a water-source heat pump, however, the objective of this adapt the model to be able to operate as an air-source heat pump. The EIR formulation will remain the same as the current model, as shown in the equation.
+
+```CapacityMultiplier(temperature) = a + b*(T_load) + c*(T_load^2) + d*(T_source) + e*(T_source^2) + f*(T_load*T_source)```
+
+This current project will rename this existing object to allow for water source and air source operation. It will also adapt the input structure of the exisiting object so that no new objects are created.
+
+## Approach
+
+- Rename the existing EIR-WWHP from:
+
+  ```HeatPump:WaterToWater:EIR:(Cooling/Heating)```
+
+  to:
+
+  ```HeatPump:PlantLoop:EIR:(Cooling/Heating)```
+
+  "PlantLoop" could also be replaced by suggestions from the reviewers. Perhaps, "WaterSide" or "WaterSideLoad"? 
```

Comment:
It is really hard to come up with a better name.  Unless we just want to take out the middle descriptor, leaving out plant loop entirely.  But then it doesn't really describe where it should be used.  In the very long run, if a single heat pump could be used for providing heating and cooling to both air and water, then we could just have HeatPump:EIR...but for now this seems fine.